### ISSUE TITLE
mark assembly as CLSCompliant

### ISSUE THREAD

**Issue by SurajGupta** [dotnet-state-machine/stateless#92]
If possible, could stateless be marked as CLSCompliant in AssemblyInfo?   This enables CLSCompliant code to take a dependency on stateless without a compiler warning.

**Comment by nblumhardt** [comment]
Hi Suraj - yes, this would be fine. Are you interested in submitting a PR with the change? Thanks!

**Comment by SurajGupta** [comment]
Happy to.  I'll submit that in a second.  Note however that the project doesn't build out-of-the-box.  I have a brand new box with fresh install of the latest VS 2015 and I'm getting the following errors when opening the solution.  So I haven't actually been able to compile the PR I'm sending - but it's a trivial fix.

```
D:\src\stateless\Stateless\Stateless.xproj : error  : The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\DotNet\Microsoft.DotNet.Props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.  D:\src\stateless\Stateless\Stateless.xproj

D:\src\stateless\Stateless.Tests\Stateless.Tests.xproj : error  : The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\DotNet\Microsoft.DotNet.Props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.  D:\src\stateless\Stateless.Tests\Stateless.Tests.xproj

D:\src\stateless\BugTrackerExample\BugTrackerExample.xproj : error  : The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\DotNet\Microsoft.DotNet.Props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.  D:\src\stateless\BugTrackerExample\BugTrackerExample.xproj

D:\src\stateless\TelephoneCallExample\TelephoneCallExample.xproj : error  : The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\DotNet\Microsoft.DotNet.Props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.  D:\src\stateless\TelephoneCallExample\TelephoneCallExample.xproj

D:\src\stateless\OnOffExample\OnOffExample.xproj : error  : The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\DotNet\Microsoft.DotNet.Props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.  D:\src\stateless\OnOffExample\OnOffExample.xproj

D:\src\stateless\BugTrackerExample\BugTrackerExample.xproj : error  : The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\DotNet\Microsoft.DotNet.Props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.  D:\src\stateless\BugTrackerExample\BugTrackerExample.xproj
```

**Comment by SurajGupta** [comment]
here you go!
https://github.com/dotnet-state-machine/stateless/pull/93

**Comment by HenningNT** [comment]
I had to install the net.core before being able to open the project, have done that?

**Comment by SurajGupta** [comment]
Will try and will update the README if it works.

**Comment by SurajGupta** [comment]
Worked!  I updated the README, closed the pull request, and opened another: https://github.com/dotnet-state-machine/stateless/pull/95

**Comment by nblumhardt** [comment]
Merged in #95
